img/f/fairphone.com.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 177.2 24"><path d="M24 14.6a195 195 0 0 1 2.8-6.8l1.4 6.8zM24.9.3a338 338 0 0 0-10.3 23.3h4.8c.7 0 1.3-.5 1.6-1.1l1.1-3h7.1l.8 3c.2.7.8 1.1 1.4 1.1h4.9c-2.2-7.8-4-15.5-5.6-23.3Zm14.3 23.3H45c.5-7.7 1.2-15.4 2.4-23.1h-5.8c-1.1 7.6-1.9 15.4-2.4 23.1M63.1 8.7c-.2 1.9-1.5 3-3.5 3h-2.9l.7-6.1h2.8c2 0 3.1 1 2.9 3.1 0-.1 0-.1 0 0m5.8-.3v-.2c.3-2.3-.2-4.1-1.3-5.4C66.4 1.3 64.2.5 61.2.5H50.8l-.7 5.2h1.6c-.8 6-1.3 12-1.7 18h5.8l.5-7.2h2.1l2.9 6.3c.2.5.8.8 1.3.8h5.6l-4-8.4a8.2 8.2 0 0 0 4.7-6.8m15.5.4c-.2 2-1.6 3.2-3.6 3.2h-2l.7-6.4h2c2.1 0 3.2 1 2.9 3.2M82.5.4h-9.7l-.7 5.2h1.6c-.8 6-1.3 11.9-1.7 18h5.8l.5-6.7h2.2c4.9-.1 9-2.7 9.7-8.3v-.1c.6-5.2-2.4-8.1-7.7-8.1m22.4 8.8h-6.4L99.6.4h-5.8c-1.1 7.7-1.9 15.4-2.4 23.1h5.8l.7-8.9h6.4l-.7 8.9h5.8c.5-7.7 1.2-15.4 2.4-23.1H106l-1.1 8.8m44.8 2.9c-2.2-4-4.3-8-6.2-12l-5.3.9c-1.1 7.6-1.9 15.1-2.3 22.7h5.7c.2-4 .6-8.1 1-12.1 2 3.8 4.2 7.6 6.4 11.4.3.4.7.7 1.2.7h4.4A226 226 0 0 1 157 .6h-5.7c-.7 3.7-1.2 7.6-1.6 11.5M6.9 9.9l.5-4.1H18l.7-5.4H.8L0 5.8h1.6C.9 11.7.3 17.6 0 23.6h5.8l.6-8.6h6.2c.9 0 1.7-.7 1.7-1.6l.4-3.5zm158.5 8.4.3-4h6.3c.9 0 1.7-.7 1.8-1.6l.3-3.3h-7.9l.5-3.8h9.8l.7-5.2h-17.1l-.7 5.2h1.6c-.8 6-1.3 11.9-1.7 17.9H175l.4-5.2zm-37.5-5.8v.4c-.4 4-2.2 5.8-4.9 5.8s-4.4-2.4-4-6.4v-.4c.4-4 2.6-6.4 5.2-6.4 1.3 0 2.3.6 3 1.7l-.8.8a6 6 0 0 1 1.5 4.5m3.1-8.9.8-.8a8.7 8.7 0 0 0-6.7-2.8c-6 0-11.2 4.9-11.9 11.8l-.1.5c-.7 6.9 3.4 11.7 9.5 11.7s10.5-4.2 11.2-11.1l.1-.5c.3-3.7-1.1-6.7-2.9-8.8" fill="#2fa6df"/></svg>
```

Comment:
This will hardly be readable in a small size. Maybe just the _F_ would be sufficient?
![](https://raw.githubusercontent.com/phallobst/phallobst/main/resterampe/fairphone.com.svg)
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" width="23.2" height="23.2"><path d="m9.15 9.5.5-4.1h10.6l.7-5.4H3.05l-.8 5.4h1.6c-.7 5.9-1.3 11.8-1.6 17.8h5.8l.6-8.6h6.2c.9 0 1.7-.7 1.7-1.6l.4-3.5z" style="fill:#2fa6df"/></svg>
```